guides/plugins/apps/payment.md

```diff
@@ -40,6 +40,10 @@ Depending on the URLs you provide, Shopware knows which kind of payment flow you
 
 ## Synchronous payments
 
+::: info
+Be aware, that from Shopware 6.7.0.0 onwards your app-server **has to** respond with a payment state in its response.
```

Comment:
untrue, only, if you want to set it, there is just no default state transition anymore.

Reply:
Also, shouldn't this info be put into the main branch?

Reply:
This will be in the main branch of course, but I thought I let it here for good measure ?